### ISSUE TITLE
bug(ci): deadlink checker didn't find the wrong url in sidebar

### ISSUE THREAD

**Issue by seeflood** [mosn/layotto#799]
**What happened**:
@wenxuwan found that the link in the sidebar is wrong, see https://mosn.io/layotto/#/zh/design/oss/oss-api-design
![image](https://user-images.githubusercontent.com/26001097/192236479-76b9f61c-c156-4f9b-938f-7646c631731d.png)

The problem is that our deadlink checker didn't find it :(
The checker runs successfully without noticing this link:
https://github.com/mosn/layotto/actions/workflows/dead-link-checker.yaml
![image](https://user-images.githubusercontent.com/26001097/192236818-e9862cd3-2ae3-466f-b4ca-ef64b6faa484.png)

**What you expected to happen**:
deadlink checker should find out the wrong urls

**How to reproduce it (as minimally and precisely as possible)**:
open https://mosn.io/layotto/#/zh/design/oss/oss-api-design

**Anything else we need to know?**:

**Comment by seeflood** [comment]
cc @Xunzhuo

**Comment by seeflood** [comment]
I found that `curl -w %{http_code}  https://mosn.io/layotto/#/zh/design/oss/oss-api-design` returns 200, not 404 
Ok. It's docsify's problem.   :(
This issue is not a high priority

**Comment by rayowang** [comment]
This looks like a routing problem with webserver configuration.

**Comment by seeflood** [comment]
Yes, but I don't know how to configure the docsify server to fix this issue 😢
